src/components/NavBar.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const NavBar = () => {
+  return <div>NavBar</div>;
+};
```

Comment:
- [ ]  this `Navbar` can be usable 👍  Add `Navlink`  by importing from `react-router-dom` and give them a specific path and React router provides for us the route changers components that we can use to have a smooth navigation. The <NavLink> components. 👍 😄